hugo/layouts/partials/home/our-links.html

```diff
@@ -1,10 +1,6 @@
-<p>
-    {{ $home := $.Site.Data.home.home }}
-    {{ range where $home.hero.links "url" "ne" "#"  }}
```

Comment:
But we do have links for those 2 buttons. Should we consider the possibility of not having a link for this in the future?